include/open62541/server_config.h

```diff
@@ -166,6 +167,13 @@ struct UA_ServerConfig {
     /* Nodestore */
     UA_Nodestore nodestore;
 
+#ifdef UA_ENABLE_USE_ENCODED_NODES
```

Comment:
Move this inside the binary nodestore plugin.
Do not make changes in this file.